src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/BDD.cs

```diff
@@ -423,6 +413,19 @@ long Get(int i)
 
             //the result is the last BDD in the nodes array
             return nodes[n - 1];
+
+            // Gets the i'th element from the underlying array of longs represented by bytes
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
When we place this attribute, do we have an expectation that someone has verified that it's not otherwise getting inlined (and possibly considered opening a bug against codegen if it "obviously" ought to be)?

Reply:
(vs simply - it is clearly profitable, and I'm not certain it will happen)

Reply:
We generally use it only if the overhead matters and there are appropriate tradeoffs around potential code growth, and we validate it isn't ever inlined or it's critical it does and it's borderline whether it is inlined or not.